convert/jenkinsjson/json/nexusArtifactUploader.go

```diff
@@ -0,0 +1,77 @@
+package json
+
+import (
+	"encoding/json"
+	harness "github.com/drone/spec/dist/go"
+	"log"
+)
+
+const (
+	ArtifactUploaderImage = "harnesscommunity/drone-nexus-publish"
```

Comment:
```suggestion
	ArtifactUploaderImage = "plugins/nexus-publish:latest"
```

modify it in other places accordingly